envbuilder.go

```diff
@@ -342,6 +342,14 @@ func Run(ctx context.Context, options Options) error {
 		}
 	}
 
+	if options.PushImage {
+		// Make sure the Dockerfile is using the correct directives to run envbuilder
+		buildParams.DockerfileContent = buildParams.DockerfileContent + "\n" +
```

Comment:
I guess that you'll have a merge conflict with the other PR: https://github.com/coder/envbuilder/issues/216